### ISSUE TITLE
[Bug] TS2304: Cannot find name 'URL'

### ISSUE THREAD

**Issue by galkin** [Urigo/graphql-scalars#109]
```
> tsc --project tsconfig.json

node_modules/graphql-scalars/dist/esnext/mocks.d.ts:14:30 - error TS2304: Cannot find name 'URL'.

14 declare const URLMock: () => URL;
```

At application level, it can be fixed by
1) adding `dom` to `compilerOptions.lib` array
2) copy pasting URL from typescript/lib/lib.dom.d.ts

Could you fix this bug on package level?

**Comment by ardatan** [comment]
Available in 1.0.2!

**Comment by galkin** [comment]
@ardatan, thank you for fast fix. Could you check related regressionbug https://github.com/Urigo/graphql-scalars/issues/112 ?
